worlds/sc2/docs/setup_fr.md

```diff
@@ -0,0 +1,214 @@
+
+La page [*datapackage*](/datapackage) d'Archipelago liste l'ensemble des *items* et des *locations* de tous les jeux 
+que le site web prend en charge actuellement, dont ceux de *StarCraft 2*.
+
+Vous trouverez aussi la liste complète des *items* de *StarCraft 2 Archipelago* à la page 
+[*Icon Repository*](https://matthewmarinets.github.io/ap_sc2_icons/). 
+Notez que cette page contient diverses informations supplémentaires sur chacun des *items*.
+Cependant, l'information présente dans cette dernière peut différer de celle du *datapackage* d'Archipelago puisqu'elle 
+est générée, habituellement, à partir des développements récents de *StarCraft 2 Archipelago* qui n'ont peut-être pas 
+encore été inclus dans le site web d'Archipelago.
+
+## Comment est-ce que je peux joindre un *MultiWorld*?
+
+1. Exécuter `ArchipelagoStarcraft2Client.exe`.
+   - Uniquement pour cette étape, les utilisateurs de macOS devraient plutôt suivre les instructions à la page 
+["Exécuter sous macOS"](#exécuter-sous-macos).
+2. Entrer la commande `/connect [server ip]`.
+   - Si le *MultiWorld* est hébergé via un siteweb, l'IP du server devrait être indiqué dans le haut de la page de 
+votre *room*.
+3. Inscrivez le nom de joueur spécifié dans votre *yaml* lorsque vous y êtes invité.
+4. Si le serveur a un mot de passe, l'inscrire lorsque vous y êtes invité.
+5. Une fois connecté, aller sur l'onglet *StarCraft 2 Launcher* dans le client. Dans cet onglet, vous devriez trouver 
+toutes les missions de votre monde. Les missions qui ne sont pas disponibles présentement auront leur texte dans une 
+tonte de gris. Vous n'avez qu'à cliquer une des missions qui est disponible pour la commencer!
+
+## *StarCraft 2* ne démarre pas quand je tente de commencer une mission
+
+Pour commencer, regarder le fichier *log* pour trouver le problème (ce dernier devrait être dans 
+`[Archipelago Directory]/logs/SC2Client.txt`).
+Si vous ne comprenez pas le problème avec le fichier *log*, visitez notre 
+[*Discord*](https://discord.com/invite/8Z65BR2) pour demander de l'aide dans le forum *tech-support*.
+Dans votre message, veuillez inclure une description détaillée de ce qui ne marche pas et ajouter en pièce jointe le 
+fichier *log*.
+
+## Mon profil de raccourcis clavier n'est pas disponibles quand je joue à *StarCraft 2 Archipelago*
+
+Pour que votre profil de raccourcis clavier fonctionne dans Archipelago, vous devez copier votre fichier de raccourcis 
+qui se trouve dans `Documents/StarCraft II/Accounts/######/Hotkeys` vers `Documents/StarCraft II/Hotkeys`.
+Si le dossier n'existe pas, créez-le.
+
+Pour que *StarCraft 2 Archipelago* utilise votre profil, suivez les étapes suivantes.
+Lancez *Starcraft 2* via l'application *Battle.net*. 
+Changez votre profil de raccourcis clavier pour le mode standard et acceptez, puis changez pour votre profil 
+personnalisé et acceptez. 
+Vous aurez juste besoin de faire ça une seule fois.
```

Comment:
"Vous n'aurez besoin de faire ça qu'une seule fois" sounds a bit better, but this is a nitpick, both work.

Reply:
Applied as suggested!